demisto_sdk/commands/test_content/test_modeling_rule/init_test_data.py

```diff
@@ -0,0 +1,152 @@
+import logging
+import traceback
+from io import StringIO
+from pathlib import Path
+from typing import List, Set
+
+import typer
+
+from demisto_sdk.commands.common.content.objects.pack_objects.modeling_rule.modeling_rule import \
+    ModelingRule
+from demisto_sdk.commands.common.logger import setup_rich_logging
+from demisto_sdk.commands.test_content.xsiam_tools.test_data import (EventLog,
+                                                                     TestData)
+
+app = typer.Typer()
+logger = logging.getLogger('demisto-sdk')
+
+
+@app.command(no_args_is_help=True)
+def init_test_data(
+    input: List[Path] = typer.Argument(
+        ...,
+        exists=True,
+        dir_okay=True,
+        resolve_path=True,
+        show_default=False,
+        help=('The path to a directory of a modeling rule. May pass multiple paths to '
+              'initialize test data files for multiple modeling rules.')
+    ),
+    count: int = typer.Option(
+        1,
+        '-c', '--count',
+        min=1,
+        show_default=True,
+        help='The number of events to initialize the test data file for.',
+    ),
+    verbosity: int = typer.Option(
+        0,
+        '-v', '--verbose',
+        count=True,
+        clamp=True,
+        max=3,
+        show_default=True,
+        help='Verbosity level -v / -vv / .. / -vvv',
+        rich_help_panel='Logging Configuration'
+    ),
+    quiet: bool = typer.Option(
+        False,
+        help='Quiet output - sets verbosity to default.',
+        rich_help_panel='Logging Configuration',
+    ),
+    log_path: Path = typer.Option(
+        None,
+        '-lp', '--log-path',
+        resolve_path=True,
+        show_default=False,
+        help='Path of directory in which you would like to store all levels of logs.',
```

Comment:
Add:
If not given, then log_file_name will be disregarded, and the log output will be to stdout